supernode/daemon/mgr/SchedulerMgr.go

```diff
@@ -0,0 +1,33 @@
+package mgr
+
+import (
+	"context"
+
+	"github.com/dragonflyoss/Dragonfly/apis/types"
+)
+
+// SchedulerMgr has two main responsibilities:
+// 1. Responsible for maintaining the correspondence between peer and pieces.
+// 2. Responsible for calculating scheduling results according to certain rules.
+type SchedulerMgr interface {
+	// InitProgress init the correlation information between peers and pieces, etc.
+	InitProgress(ctx context.Context, taskID, cid string) error
```

Comment:
Could we start avoiding writing `cid` any more, people may start asking: why `taskID` is not written as `tid`. It is `clientID`, right?

Reply:
Yeah, you are right, I will change it.